### PR TITLE
Update readme.md

### PR DESCRIPTION
**Pull request by pevik**
Update `README.md` a bit.

https://github.com/pevik/iputils/tree/update-README.md

* README.md: Add removed tools
* README.md: Add history
* README.md: Reformat

### CODE REVIEW

**Comment by pevik** [comment]
@nmeyerhans could you please check grammar as a native speaker?

File: README.md [hunk 1]
```diff
@@ -12,10 +20,23 @@ These tools are included in iputils
...
-If you still use [old version](http://www.skbuff.net/iputils/), please consider moving forward to new releases placed here.
+## Tools removed from iputils
 
-This version also fully supports glibc, uClibc and musl-libc.
+Some obsolete tools has been removed (see
+[#363](https://github.com/iputils/iputils/issues/363)).
 
-Localization is hosted on [Fedora Weblate](https://translate.fedoraproject.org/projects/iputils/iputils/).
+| Tool | Removing commit | Latest release where included
+| ---- | --------------- | -----------------------------
```
Comment (pevik) [inline README.md:29]: @nmeyerhans Latest release *where* included vs. Latest release *when* included?

Comment (nmeyerhans) [reply README.md]: I might slightly prefer _when_ instead of _where_.

You might also just change the header to "Last release"

Comment (pevik) [reply README.md]: "Last release" is really the best, thanks!

**Comment by pevik** [comment]
FYI added your tag `Reviewed-by: Noah Meyerhans <noahm@debian.org>`, going to merge during today.

**Pull request closed without merging**